app/models/post.rb

```diff
@@ -1,5 +1,14 @@
 class Post < ApplicationRecord
   belongs_to :user
-  has_many :comments
-  has_many :likes
+  has_many :comments, dependent: :destroy
+  has_many :likes, dependent: :destroy
```

Comment:
This is a nice approach but it's recommended to use foreign_key: 'author_id'

Reply:
:confused: